.github/workflows/publish-release.yml

```diff
@@ -201,10 +201,10 @@ jobs:
           VERSION=$(grep -oE '\d+\.\d+' <<< "${{ github.ref_name }}" | { read version; echo "$version-stable"; })
           echo "VERSION=$VERSION" >> $GITHUB_OUTPUT
 
-          curl -L -X POST https://api.github.com/repos/react-native-community/template/release.yaml/dispatches \
+          curl -L https://api.github.com/repos/react-native-community/template/actions/workflows/release.yaml/dispatches
```

Comment:
Wouldn't it be safer to use the client library? https://github.com/octokit/plugin-rest-endpoint-methods.js